src/pimcore/scss/_gui.scss

```diff
@@ -0,0 +1,266 @@
+// stylelint-disable selector-class-pattern
```

Comment:
Personally, I would keep pimcore specific files out of the vue boilerplate, but if you want to have it, it's fine.